README.md

```diff
@@ -147,8 +158,7 @@ the version number to simplify deployment to WebLogic Server.
 
 # Sidecar
 
-The sidecar is a standalone process that runs the exporter. It is primarily intended for use by the
```

Comment:
The 'web-app' section above fails to mention the coordination configurator, and is imprecise about the usage of the web-form. It should be updated to match the contents of the introduction.

Reply:
It's in the intro:
```
You may include a configuration file directly in the WAR file, and you may temporarily modify the configuration in a 
 running system by using a web form. If a [coordination configurator](config_coordinator/README.md) is running and configured, 
 that temporary configuration will be sent to all servers configured to use it.
 ```

Reply:
The 'web-app' section is a reference for the 'web-app', it should be complete.  In general, documentation about a specific feature should be relatively complete in of itself, even if a part of the feature is described in an intro/summy.